packages/addresses/src/address.ts

```diff
@@ -1,9 +1,23 @@
-import { base58, Serializer, string } from '@metaplex-foundation/umi-serializers';
+import { Codec, combineCodec, Decoder, Encoder, mapEncoder } from '@solana/codecs-core';
+import { getBase58Decoder, getBase58Encoder, getStringDecoder, getStringEncoder } from '@solana/codecs-strings';
 
 export type Base58EncodedAddress<TAddress extends string = string> = TAddress & {
     readonly __brand: unique symbol;
 };
 
+let memoisedBase58Encoder: Encoder<string> | undefined;
+let memoisedBase58Decoder: Decoder<string> | undefined;
+
+function getMemoisedBase58Encoder(): Encoder<string> {
```

Comment:
Nit: should this be "memoized" with a Z since I'm assuming we're following American English in the other libraries?

Reply:
Yes, but I don't like it :p Fixed, TY!